deps/build.jl

```diff
@@ -121,60 +139,42 @@ if any(isfile.(hsl_archives))
     write(io, "\n")
   end
 
-  if isfile(hsl_ma57_archive)
-    @info "building ma57"
-    path_libhsl_ma57 = joinpath(libdir, "libhsl_ma57.$dlext")
-    open(path_deps, "a") do io
-      write(io, "const libhsl_ma57 = \"$path_libhsl_ma57\"\n")
-    end
-    if isfile(hsl_ma57_patch)
+  for software in keys(hsl_collection)
+    if haskey(hsl_archives, software) && isfile(hsl_archives[software])
+      @info "building $(software)"
```

Comment:
we're not really building here, are we?

Reply:
We are building here but the `build` function is at the end of the "if" structure.